src/include/structs.h

```diff
@@ -100,6 +101,7 @@ struct s_FlagInfo {
 struct s_StructureInfo {
     // Structures
     SDL_Surface *bmp;         // a pointer to the 8 bit bitmap
+    Texture *shadow;	   // a pointer to the shadow bitmap
```

Comment:
Trying to understand this, why does this need to be a texture? And not an `SDL_Surface` ?